### DISCUSSION TITLE
Plugin Stopped Working on Corporate machine

### DISCUSSION THREAD

**Discussion by CarolusMagnuss** [joyfullservice/msaccess-vcs-addin#346] [Q&A]
Hello again,

I have been working with the plugin for while now and it is an essential tool for Access development in my opinion. However after an Office Update on Monday it stopped working on my corporate machine, prompting the error: 

Error: Loading options from C:\\Users\\Username\\Appdata\\Roaming\\msaccessvcs\vcs-options.json

Error -21477221164: class not registered Source: clsOptions.LoadOptionsFromFile

I suspect either one of the Microsoft Security Patches or a Corporate policy which blocks MS-Access PlugINs. Do you have any idea what is the source of the issue?

Regards

**Comment by hecon5** [comment]
Some of our devs get that fairly often...the way we workaround it is to reinstall the addin (sometimes this fixes it). 

Failing that, we'll add the addin as a reference, run export, and then remove it as a reference, restart the db, and poof. it starts working fine again. ?why? it does this: no clue. but it's been reasonably successful in getting it to play nice again.

**Comment by mwolfe02** [comment]
What version (and build number) of Access are you using where you are seeing the problem?  

Did you get a chance to follow @hecon5's troubleshooting tips?

I posted about this issue on the Access MVP group mailing list.  One of the Access devs tried to reproduce the problem but was unable to.  Another Access MVP asked whether this recent VBA issue could be related: [Error when trying to open an ACCDE/MDE file created in a different version of Access](https://support.microsoft.com/en-us/topic/error-when-trying-to-open-an-accde-mde-file-created-in-a-different-version-of-access-f4cd36cd-549e-42ba-b75a-dfe964294a81).
